1-run-automation-test.md

```diff
@@ -0,0 +1,123 @@
+ + For instruction on how to install and configure TeamCity server, follow [this guide](https://confluence.jetbrains.com/display/TCD18/Installing+and+Configuring+the+TeamCity+Server).
+
+Set up your project and link your app repository that we will test with Kobiton later.
++ Integrate GitHub with TeamCity, you can follow [this guide](https://confluence.jetbrains.com/display/TCD10/Integrating+TeamCity+with+VCS+Hosting+Services).
+ 
+> For more information about TeamCity can be found in [this tutorial](https://confluence.jetbrains.com/display/TCD10/TeamCity+Documentation).
+
+## 2. Config automation test script
+This documentation uses NodeJS as the default language to demonstrate the guide. 
+### 2.1 Get Username and API key
+Go to https://portal.kobiton.com.
+- Username
+  + Select *user Icon* -> *Profile* (you might find it in the top right corner)
+
+  ![alt text](./assets/username.png )
+
+- API key 
+  + Select *user Icon* -> *Settings* (you might find it in the top right corner)
+
+  ![alt text](./assets/api_key.png )
+
+### 2.2 Get desired capabilities
+The desired capabilities need to be added to the automation test script to allow the test to be executed on Kobiton device.
+  + Click *Devices* in the top navigation bar.
+ 
+    ![alt text](./assets/devices.png )
+ 
+  + Hover over any device you want to test with and click on the Automation settings button. A dialog box will then appear on screen.
+ 
+    ![alt text](./assets/click_auto.png )
+ 
+  + On the left hand side, you can select your preferred language, App Type, etc.
+     > For this example, we use `NodeJS` as the language to run automation test. 
+
+    ![alt text](./assets/automation.png ) 
+### 2.3 Config TeamCity project
+Follow steps below to setup TeamCity for automation testing
++ In your Teamcity project, on the left side of the page, click *Build Step* to set build steps.
++ On the right side of the page, for the *Runner type* category, choose *Command Line*.
++ Add below commands to `Custom script` part.
+
+```
+  cd javascript
+  npm install
+  npm run android-app-test
+```
+> Note:
+> + `cd javascript` is the command if you using NodeJS
+> + `npm install` : install missing dependencies
+> + `npm run android-app-test` : run the test script
+
+![alt text](./assets/cmd.png)
+
+**(Optional) Attaching environment variable to Teamcity**
++ In your project in TeamCity, on the left side of the page, click *Parameters*. Then click `Add new parameter` button. 
+
+![alt text](./assets/param.png)
+
++ Add your Kobiton username and API key.
+
+![alt text](./assets/param_name.png)
+
+![alt text](./assets/param_key.png)
+### 2.4 Config automation test script
+Kobiton has already provided sample scripts for automation testing, visit [here](https://github.com/kobiton/samples) for reference.
+
+Replace the Kobiton `username`, `apiKey` and `desiredCaps` that collected from above part to the script.
```

Comment:
I think that it should be replaced with 'replace values of username, apiKey and desiredCaps in the script with the ones collected above' will make it more clear and easier to understand.